conan/test_package/testsuite-support/src/dynamic_config.cpp

```diff
@@ -0,0 +1,34 @@
+#include "dynamic_config.hpp"
```

Comment:
This `testsuite-support` test will be replaced with a much more compact one, according to the discussion in PM